cnxepub/models.py

```diff
@@ -337,12 +337,14 @@ def insert(self, i, v):
 
 class Binder(TranslucentBinder):
     """An object that has metadata and contains
-    ``Binder``, ``TranslucentBinder`` and ``Document`` instances.
+    ``Binder``, ``Resource``, ``TranslucentBinder`` and ``Document`` instances.
```

Comment:
Technically, it doesn't "contain" Resources the same way. In this we are talking about the containment as in contained within a sequence and therefore useful for the ``__contains__`` method.